skipscale/main.py

```diff
@@ -16,7 +16,11 @@ async def healthcheck(request):
     return Response(status_code=200)
 
 routes = [
+    # Used for original images
     Route('/original/{tenant}/{image_uri:path}', original),
+    # Used for News assets
```

Comment:
Would not refer to News here, instead explain that it's for non-image assets that traverse skipscale just for the crypto infrastructure